addons/nametags/stringtable.xml

```diff
@@ -110,16 +110,16 @@
             <Portuguese>Cor padrão do nome (unidades fora do grupo)</Portuguese>
         </Key>
         <Key ID="STR_ACE_NameTags_Module_DisplayName">
-            <English>Name Tags</English>
-            <Polish>Ustawienia imion</Polish>
-            <Spanish>Etiquetas de nombre</Spanish>
-            <German>Zeige Spielernamen</German>
-            <Czech>Jmenovky</Czech>
-            <Portuguese>Etiquetas de nome</Portuguese>
-            <French>NameTags</French>
-            <Hungarian>Névcímkék</Hungarian>
-            <Russian>Имена игроков</Russian>
-            <Italian>Etichette Nomi</Italian>
+            <English>ACE Name Tags</English>
```

Comment:
It's the same as keybinds. The idea is that every mod has a category. ACE is so big that components should be grouped together. Nametags has enough settings to get one tab itself.
I suggest you check it out ingame.
To view the missions tab (which has 3 additional options formly tagged with `clientSettable = 0`) just enter 3den. The menu can be found under "Settings".